ios/Runner/AppDelegate.swift

```diff
@@ -1,13 +1,27 @@
 import UIKit
 import Flutter
+import workmanager
 
 @UIApplicationMain
 @objc class AppDelegate: FlutterAppDelegate {
   override func application(
     _ application: UIApplication,
     didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?
   ) -> Bool {
+    if #available(iOS 10.0, *) {
+      UNUserNotificationCenter.current().delegate = self as? UNUserNotificationCenterDelegate
+    }
     GeneratedPluginRegistrant.register(with: self)
+    WorkmanagerPlugin.registerTask(withIdentifier: "com.cBreez.paymentreceived")
+    UIApplication.shared.setMinimumBackgroundFetchInterval(TimeInterval(60*15))
```

Comment:
Why do we need background fetch? The trigger for working in the background is silent push